src/Compilers/CSharp/Portable/Compiler/TypeCompilationState.cs

```diff
@@ -93,7 +93,7 @@ public NamedTypeSymbol Type
             get
             {
                 // NOTE: currently it can be null if only private implementation type methods are compiled
```

Comment:
This comment seems wrong now?

Reply:
No, just that this file was annotated incorrectly previously.  Will fix it.

---
In reply to: [392366627](https://github.com/dotnet/roslyn/pull/42068#discussion_r392366627) [](ancestors = 392366627)